src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Controllers/AuthenticationControllerTests.cs

```diff
@@ -1519,7 +1523,7 @@ private static bool HasCookieValue(IEnumerable<string> setCookieHeaders, string
             Assert.NotNull(responseTypeParam);
             Assert.NotNull(clientIdParam);
             Assert.Equal(HttpUtility.UrlDecode(expectedRedirectUri), redirectUriParam);
-            Assert.Equal("openid", scopeParam);
+            Assert.Contains("openid", scopeParam);
```

Comment:
The Assert.Contains() method expects an IEnumerable<T>. If it works it works, but it looks wrong to use the string as a type of collection.